try:
    result=45/0
    
except:
    print("error")
finally:
    print('the programme continue')

print('done')